dev_process.md

```diff
@@ -0,0 +1,86 @@
+# Development Process
+One of the most crucial factors in having a secure codebase is a solid development process. "An ounce of prevention is worth a pound of cure". This document gives an *example* development process that we, at Nascent, have found works well. In addition to getting better overall code quality, this process serves as a companion to our [audit readiness checklist](https://github.com/nascentxyz/nascent-security/blob/main/audit-readiness-checklist.md). If you follow the below process, you should naturally check off most of the items on the checklist.
+
+```
+Feature Request received
+ |
+ └> Specification Write up
+     |
+     └> Evaluation (Time + complexity + risks)
+        |
+        └> Implementation
+           |
+           └> Test
+              |
+              └> Deployment
+                 |
+                 └> Monitor
+```
+## Specification
+Make note of the kinds of variables that affect the feature:
+1. User input?
+2. Time?
+3. Other protocols?
+4. Existing state?
+
+## Note on Evaluation
+Make dedicated time to evaluate:
+1. How long will this realistically take? Without specification most estimates will be wrong.
+2. Is the specification overly complex? Complexity leads to bugs and worse overall code
+3. What are the risks associated with this feature? Spend ample time evaluating this. Consider every module the feature may touch. Then go back through excluded modules and *ensure* they cannot be affected.
+
+Now that you have millions of dollars at risk, you have to consider each feature may put *all* of it at risk. The cost of a misstep will likely be in the millions of dollars. 
```

Comment:
This assumes post-launch? Overall, this is one of a few places where it seems like we're talking about making upgrades to an existing contract rather than initial protocol devleopment.

Reply:
Ye will re-word to make it more generic. I don't want to leave out post-launch process but this was poorly worded